Scripts/script-Base64ListToFile.yml

```diff
@@ -13,12 +15,17 @@ script: |-
 
   # get the list
   res = demisto.executeCommand('getList', {'listName': demisto.args()['listname']})
+  if res['Type'] == entryTypes['error']:
+    return_error("error reading list %s from demisto" % demisto.args()['listname'])
+    sys.exit(0)
```

Comment:
Please remove `sys.exit(0)` as `return_error` already execute that

Reply:
ok